core/transaction_builder/helper/create_objects_helper.cpp

```diff
@@ -119,14 +119,15 @@ Api::Account createAccount(std::string publicKey, std::string name,
   return ret;
 }
 
-Api::Asset createAsset(std::string domain, std::string name,
-                       ::txbuilder::Map value, std::string smartContractName) {
+Api::Asset createAsset(
+  std::string domain,
+  std::string name,
+  ::txbuilder::Map value, std::string smartContractName) {
   Api::Asset ret;
   ret.set_domain(std::move(domain));
   ret.set_name(std::move(name));
   *ret.mutable_value() = ::google::protobuf::Map<std::string, Api::BaseObject>(
       value.begin(), value.end());
-  ret.set_smartcontractname(std::move(smartContractName));
```

Comment:
I want to confirm this branch and develop as v0.75.
And thia version doesn't need smart contract.
so I remove it temporarily